net-p2p/qbittorrent/qbittorrent-4.5.5-r2.ebuild

```diff
@@ -12,30 +12,28 @@ if [[ ${PV} == *9999 ]]; then
 	EGIT_REPO_URI="https://github.com/qbittorrent/qBittorrent.git"
 	inherit git-r3
 else
-	SRC_URI="https://github.com/qbittorrent/qBittorrent/archive/release-${PV}.tar.gz -> ${P}.tar.gz"
+	SRC_URI="
+		mirror://sourceforge/qbittorrent/${P}.tar.xz
+		verify-sig? ( mirror://sourceforge/qbittorrent/${P}.tar.xz.asc )
+	"
 	KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~riscv ~x86"
-	S="${WORKDIR}"/qBittorrent-release-${PV}
+
+	BDEPEND="verify-sig? ( sec-keys/openpgp-keys-qbittorrent )"
+	VERIFY_SIG_OPENPGP_KEY_PATH=/usr/share/openpgp-keys/qBittorrent.asc
 fi
 
 LICENSE="GPL-2"
 SLOT="0"
-IUSE="+dbus +gui qt6 systemd test webui"
+IUSE="+dbus +gui qt6 test webui systemd"
 RESTRICT="!test? ( test )"
-REQUIRED_USE="|| ( gui webui )"
+REQUIRED_USE="|| ( gui webui )
```

Comment:
I'd perhaps consider consistent:
```
REQUIRED_USE="

"
```
style, like other variables.

One just to be consistent, two because it makes diffs look a bit nicer. But not a blocker at all obviously.